README.md

```diff
@@ -32,14 +32,15 @@ More information on terminology can be found [here](docs/terminology.md).
 To populate that database
 ```
 ## Initialize DBs
-./phg initdb --dbpath /path/to/dbs
+./phg initdb --db-path /path/to/dbs
 
 ## Build VCF data
 ./phg create-ranges --gff my.gff --boundary gene --pad 500 -o /path/to/bed/file.bed
-./phg align-assemblies --gff anchors.gff --ref Ref.fa -a assembliesList.txt -o /path/to/temp
+./phg align-assemblies --gff anchors.gff --reference-file Ref.fa -a assembliesList.txt -o /path/to/temp
```

Comment:
Is this still `temp`?

Reply:
Nope, changed this as part of Lynn's request as well.